src/main/java/org/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -145,7 +147,11 @@ private void removeAbbreviation() {
 
     @FXML
     private void saveAbbreviationsAndCloseDialog() {
-        viewModel.saveEverythingAndUpdateAutoCompleter();
+        viewModel.save();
+
+        // Update journal abbreviation loader
+        Globals.journalAbbreviationRepository = JournalAbbreviationLoader.loadRepository(preferences.getJournalAbbreviationPreferences());
```

Comment:
I do not really understand this line. Why is this necessary?

Reply:
If the user changes the abbreviations, these changes must be reflected in the global repository.